workshop/exercise-2/README.md

```diff
@@ -63,4 +63,119 @@ Before your continue, make sure all the pods are deployed and **`Running`**. If
 
 Congratulations! You successfully installed Istio into your cluster.
 
+
+## Validating Installation
+
+In this exercise, you will validate your Istio installation and launch some debugging containers to experiment with the service mesh.
```

Comment:
We can't use alpine because we need more utilities than a base alpine image. I generally just keep my own (admittedly fat) utilities container around because I'm always wanting more debug tools.

The container can live in the ibmcom namespace, no problem. I don't have credentials for that so someone else will have to give me access or do the work.

I like doing it this way because then 'seeing istio work' and proving it works to yourself happens immediately. Doing it later on with guestbook would work, but that's several exercises away (after configuring istio-ingressgateway at least) and I wanted to get this in at the beginning.

I actually wanted this to be exercise-3 and everything then shifts down, but that would have made for an ugly PR.

Reply:
istio now provides a verify-install cmd in 1.1 (to be released in 2 weeks).  I'd prefer us to use that cmd when it is ready.